docs/sources/flow/reference/components/discovery.docker.md

```diff
@@ -35,18 +47,17 @@ The following blocks are supported inside the definition of
 Hierarchy | Block | Description | Required
 --------- | ----- | ----------- | --------
 filter | [filter][] | Filters discoverable resources. | no
-http_client_config | [http_client_config][] | HTTP client configuration for docker requests. | no
-http_client_config > basic_auth | [basic_auth][] | Configure basic_auth for authenticating to the endpoint. | no
-http_client_config > authorization | [authorization][] | Configure generic authorization to the endpoint. | no
-http_client_config > oauth2 | [oauth2][] | Configure OAuth2 for authenticating to the endpoint. | no
-http_client_config > oauth2 > tls_config | [tls_config][] | Configure TLS settings for connecting to the endpoint. | no
+basic_auth | [basic_auth][] | Configure basic_auth for authenticating to the endpoint. | no
+authorization | [authorization][] | Configure generic authorization to the endpoint. | no
+oauth2 | [oauth2][] | Configure OAuth2 for authenticating to the endpoint. | no
+oauth2 > tls_config | [tls_config][] | Configure TLS settings for connecting to the endpoint. | no
 
 The `>` symbol indicates deeper levels of nesting. For example,
-`http_client_config > basic_auth` refers to a `basic_auth` block defined inside
-an `http_client_config` block.
+`oauth2 > tls_config` refers to a `tls_config` block defined inside
+an `oauth2` block.
 
+[arguments]: #arguments
```

Comment:
Are we linking to arguments anywhere?

Reply:
yup on line 36 and 37

Reply:
Oh I seee, I would've expected the link definition to be in that arguments section rather in the blocks section, since the link is first used there.